jit/v2/JIT_v2.0.0.yaml

```
@@ -2116,7 +2083,7 @@ paths:
                         errorCodeMessage: 'portCallServiceID must be provided as part of a Vessel Status'
         '404':
           description: |
-            In case the implementer does not know of the `portCallServiceID` used in the request, it is possible for the implementer to reject the request by returning a `404` (Not Found). This could be because of a `PUT` request, for creating/initiating the **Port Call Service**, that has not finished processing or simply because the resource does not exist.
+            In case the implementer does not know of the `portCallServiceID` path parameter, it is possible for the implementer to reject the request by returning a `404` (Not Found). This could be because the resource does not exist.
```

```suggestion
            In case the endpoint does not know the `portCallServiceID`, it is possible to reject the request by returning a `404` (Not Found), because the resource does not exist.
```
Does this make it more clear? I avoid the word "implementer".

---

I have not removed the word impelemter as we use this many many other places

---

Okay, thanks.